include/CLI/App.hpp

```diff
@@ -2761,7 +2761,10 @@ class App {
         }
         int min_num = (std::min)(op->get_type_size_min(), op->get_items_expected_min());
         int max_num = op->get_items_expected_max();
-
```

Comment:
Could you add a little comment here, just like the block below, mentioning what this is here for?

Reply:
done